### ISSUE TITLE
Please "use Config;" in FFI::Platypus::Buffer

### ISSUE THREAD

**Issue by vadim-160102** [Perl5-FFI/FFI-Platypus#217]
```
>perl -MFFI::Platypus::Buffer -e 1
Use of uninitialized value $Config::Config{"archname"} in pattern match (m//) at C:/berrybrew/5.26.0_64_PDL/perl/site/lib/FFI/Platypus/Buffer.pm line 15.
```

**Comment by plicease** [comment]
I think #218 should fix this, though I wasn't able to reproduce the behavior on Strawberry Perl 5.30, and you didn't provide the version of Perl or Platypus that you are using (I can assume that you are on MSWin32 build of Perl since `archname` isn't needed on other platforms).  Can you please try version `1.08_01` and let me know if that silences the diagnostic?

**Comment by vadim-160102** [comment]
Your fix definitely solves the problem, thanks. I'm confused you say you can't reproduce the behavior, how Config.pm was loaded then? Tests are passed because Test::More loads it. I've tried portable 64-bit Strawberry versions 5.26, 5.28, 5.30, FFI::Platypus 1.07 (I see the line in question was the same from original 0.12). It's not warning as such that matters, but wrong assumed pointer size, of course. Anyway, doesn't matter now, with fix applied.

**Comment by plicease** [comment]
Great.  Thanks!

**Comment by plicease** [comment]
This fixed in production version 1.09.
